src/pokemon_animation.c

```diff
@@ -787,30 +787,30 @@ static const u8 sBackAnimationIds[] =
 static const u8 sBackAnimNatureModTable[] =
 {
     0x00, // NATURE_HARDY
```

Comment:
Shouldn't these use the nature constants?

Reply:
Woops, missed one. Will do that.